### PR TITLE
Added notes on gov cloud uri and graph endpoint

### PR DESCRIPTION
**Pull request by aiwangmicrosoft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,8 +69,8 @@ If you want to use your own application coordinates, please follow these instruc
 
 1. Open the `active-directory-wpf-msgraph-v2\App.xaml.cs` file
 1. Find the app key `ClientId` and replace the existing value with the application ID (clientId) of the `WpfApp` application copied from the Azure portal.
-
-5. Run the application from Visual Studio (Debug | Start without Debugging)
+1. If connecting to a national cloud, open the 'active-directory-wpf-msgraph-v2\MainWindow.xaml.cs' file and update string graphAPIEndpoint to the appropriate endpoint.
```
Comment (jmprieur) [inline README.md]: - please use the back quote sign around `graphAPIEndpoint` to higlight it.
- please provide a link to the endpoints

**Review by jmprieur — CHANGES_REQUESTED** [review]
Good start @aiwangmicrosoft 
proposed a few improvements

**Comment by aiwangmicrosoft** [comment]
Fixed the comments, @jmprieur can you review?

**Review by jmprieur — APPROVED** [review]
LGTM. Thanks @aiwangmicrosoft

**Pull request merged**